ansible_collections/arista/avd/roles/eos_designs/doc/l3ls-evpn/common-settings.md

```diff
@@ -213,6 +213,7 @@ platform_settings:
     lag_hardware_only: < true | false >
     feature_support:
       queue_monitor_length_notify: < true | false | default -> true >
+      interface_storm_control: < true | false | default -> true >
```

Comment:
I would vote for a new platform type to support VEOS-LAB as storm-control is valid for all HW as far as I know

Reply:
Well for documentation this is still correct and default is `true` so supported.